scrap_reason_code/models/reason_code.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2019 IBM Corp.
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ReasonCode(models.Model):
+    _name = 'reason.code'
```

Comment:
IMHO either this is a generic model and should go into a dedicated module, either you should be more specific e.g.: 'scrap.reason.code'

Reply:
Done.